src/components/abstractComponent.ts

```diff
@@ -95,7 +95,10 @@ export module Component {
     }
 
     public _requestedSpace(availableWidth : number, availableHeight: number): _SpaceRequest {
-      return {width: 0, height: 0, wantsWidth: false, wantsHeight: false};
+      return {
+        width: 0,
+        height: 0
+      };
```

Comment:
I personally like the above only if that style has the object literal fit on a single line neatly